resdk/scripts.py

```diff
@@ -209,53 +212,57 @@ def parse_annotation_file(annotation_file):
+                    'instrument_type': annotations[sample_n].get('INSTRUMENT', None),
+                    'seq_date': annotations[sample_n].get('SEQ_DATE', None),
+                }
 
-        # Single-end reads
-        else:
-            slug = 'upload-fastq-single'
-            input_['src'] = [fn]
+            # Paired-end reads
+            if annotations[sample_n]['PAIRED_END'] == 'Y' and annotations[sample_n]['FASTQ_PATH_PAIR']:
+                slug = 'upload-fastq-paired'
+                input_['src1'] = annotations[sample_n]['FASTQ_PATH'].split(',')
+                input_['src2'] = [os.path.join(genialis_seq_dir, f) for f in annotations[sample_n]['FASTQ_PATH_PAIR'].split(',')]
+                fn = input_['src1'] + input_['src2']
 
-        data = resolwe.run(slug, input_, descriptor, descriptor_schema, data_name=annotations[fn]['SAMPLE_NAME'])
+            # Single-end reads
+            else:
+                slug = 'upload-fastq-single'
+                input_['src'] = annotations[sample_n]['FASTQ_PATH'].split(',')
+                fn = input_['src']
 
-        if data:
-            uploaded_files.append(fn)
+            data = resolwe.run(slug, input=input_, descriptor=descriptor, descriptor_schema=descriptor_schema, data_name=sample_n)
 
-            presample = resolwe.api.presample.get(data=data.id)[0]
+            if data:
+                [uploaded_files.append(f) for f in fn]
 
-            if 'geo' not in presample['descriptor']:
-                presample['descriptor']['geo'] = {}
+                presample = resolwe.api.presample.get(data=data.id)[0]
 
-            organism = ORGANISMS.get(annotations[fn]['ORGANISM'].upper(), '')
-            if organism:
-                presample['descriptor']['geo']['organism'] = organism
+                if 'geo' not in presample['descriptor']:
+                    presample['descriptor']['geo'] = {}
 
-            experiment_type = EXPERIMENT_TYPE.get(annotations[fn]['SEQ_TYPE'].upper(), '')
-            if experiment_type:
-                presample['descriptor']['geo']['experiment_type'] = experiment_type
+                organism = ORGANISMS.get(annotations[sample_n]['ORGANISM'].upper(), '')
+                if organism:
+                    presample['descriptor']['geo']['organism'] = organism
 
-            resolwe.api.presample(presample['id']).patch({'descriptor': presample['descriptor'], 'presample': False})
-            # Setting 'presample': False, moves sample from presample to sample endpoint.
+                experiment_type = EXPERIMENT_TYPE.get(annotations[sample_n]['SEQ_TYPE'].upper(), '')
+                if experiment_type:
+                    presample['descriptor']['geo']['experiment_type'] = experiment_type
 
-        else:
-            print("Error uploading {}".format(fn), file=sys.stderr)
+                resolwe.api.presample(presample['id']).patch({'descriptor': presample['descriptor'], 'presample': True})
```

Comment:
@JureZmrzlikar fix this in your commit. You can omit `'presample': False` as it will be set to `True` by default. Don't forget to also remove comment in the line below.